CONTRIBUTING.rst

```diff
@@ -0,0 +1,159 @@
+Contributing to Vulture
+=======================
+
+Creating and cloning a fork
+---------------------------
+
+Fork the repository on GitHub and do the following:
+
+::
+
+    $ git clone https://github.com/jendrikseipp/vulture.git
+    $ cd vulture
+    $ git remote add origin https://github.com/USERNAME/vulture
+
+Where ``USERNAME`` is your GitHub username.
+
+Installation
+------------
+
+Installing virtualenv
+~~~~~~~~~~~~~~~~~~~~~
+
+We recommend using ``virtualenv`` to isolate the installation of vulture.
+You can read more about
+it at the `virtualenv documentation <http://virtualenv.readthedocs.org>`_.
+
+To install the ``virtualenv`` package using ``pip``, run:
+
+::
+
+    $ pip install virtualenv
+
+Once you have ``virtualenv`` installed, create your own environment(named 
+``vulture_dev``):
+
+::
+
+    $ virtualenv vulture_dev
+
+Now, whenever you work on the project, activate the corresponding environment.
+
+- On **Unix based** systems, this can be done with:
+
+  ::
+
+      $ source vulture_dev/bin/activate
+
+- And on **Windows** this is done with:
+
+  ::
+
+      $ vulture_dev\scripts\activate
+
+For deactivation, use:
+
+::
+
+    (vulture_dev)$ deactivate
+
+Installing vulture
+~~~~~~~~~~~~~~~~~~
+
+There are two ways of installing vulture. By installing releases from PyPI, 
+or directly from source.
+
+PyPI Installation
++++++++++++++++++
+
+::
+
+  $ pip install vulture
+
+Source installation
++++++++++++++++++++
+
+Navigate to your ``vulture`` clone directory, and run the following to install 
+in development mode:
+
+::
+
+  $ pip install -e .
+
+
+Installing tox
+~~~~~~~~~~~~~~
+
+Vulture uses tox for testing. You can read more about
+it at the `tox documentation <https://tox.readthedocs.io>`_.
+
+To install ``tox``, run:
+
+::
+
+  $ pip install tox
+
+Coding Standards
+----------------
+
+Creating a new branch
+~~~~~~~~~~~~~~~~~~~~~
+
+To start working on a pull request, create a new branch to work on. You 
+should never develop on your master branch because your master branch should 
+always be synchronized with the main repo’s master branch, which is challenging 
+if it has new commits. Create a branch using:
+
+::
+
+  $ git checkout -b your-new-branch
+
+Naming the branches
++++++++++++++++++++
+
+Branch names should be descriptive about the feature/issue it is working on, 
+but at the same time, be small.
+
+Commits
+~~~~~~~
+
+Each commit should be atomic and it's message should adequately describe the
+change in a clear manner.
+
+Testing
+-------
+
+Run ``tox``, using:
+
+::
+
+  $ tox
+
+And run ``vulture`` using:
+
+::
+
+  $ vulture vulture/ tests/
+
```

Comment:
We don't need to run vulture explicitly, it is already tested here:
https://github.com/jendrikseipp/vulture/blob/master/tests/test_script.py#L23-L24